modules/about-toolbox.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="about-toolbox_{context}"]
+= About `toolbox`
+
+`toolbox` is a tool that starts a container on a {op-system-first} system.
+The tool is primarily used to start a container that includes the required binaries and plug-ins that are needed to run commands such as `sosreport` and `redhat-support-tool`.
```

Comment:
We generally avoid hard wrapping. There are several places in this PR that break up lines at sentence endings. I personally think hard wrapping at sentence ends mostly mitigates the disadvantages of hard wrapping, but I think the team has taken the stance of discouraging hard wrapping in general.

Reply:
I genuinely forgot.  I guess that until I can find an join a team that uses 80, I can comply.

Reply:
I had always hardwrapped at 80 before joining Red Hat. It still feels like I'm committing a crime when there's a 500 char single line :sweat_smile: